src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -153,23 +156,33 @@ const PlanWizard: React.FunctionComponent = () => {
     strict: true,
     sensitive: true,
   });
-  const planBeingEdited =
-    plansQuery.data?.items.find((plan) => plan.metadata.name === editRouteMatch?.params.planName) ||
-    null;
+  const cloneRouteMatch = useRouteMatch<{ planName: string }>({
+    path: '/plans/:planName/clone',
+    strict: true,
+    sensitive: true,
+  });
+  let wizardMode: PlanWizardMode = 'create';
+  if (editRouteMatch) wizardMode = 'edit';
+  if (cloneRouteMatch) wizardMode = 'clone';
```

Comment:
I think I was trying to avoid SonarCloud yelling at me for nested ternaries, but we already have a ton of those in the JSX everywhere, so I really just need to find a way to turn that rule off instead :D I like this.

Reply:
If SonarCloud yells then that make me thing it's really a style issue!
No worries.